modules/ROOT/pages/dw-operators.adoc

```diff
@@ -694,9 +703,9 @@ payload  map ((user) ->
 
 === Sugar Syntax
 
-The update operator allows to write a shorter version syntax to avoid some boilerplate code by using a default variable named `$`. +
+The `update` operator enables you to use the default variable `$` instead of the "boilerplate" equivalent, which is longer.
 
-The following example uses the same Value selector example but using simpler syntax:
+The following example uses `$` to select the `age` value instead of the longer equivalent shown in comments:
```

Comment:
```suggestion
The following example binds the value of the expression `.age` to the default variable name `$` used to express the new value, instead of the longer equivalent shown in comments:
```

Reply:
done, copyedited